app/Http/helpers.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+use Illuminate\Http;
+
+function set_active_link($path, $active = '')
+{
+    return Request::is($path) ? $active : '';
+}
```

Comment:
This helpers file is mainly for views correct? Not sure if it makes sense for the file to live here... also not sure where it would live though. Maybe in app/Services. Might merge this tonight, so wouldn't worry to much about moving it, unless you think of a better spot and have time